dss/operations/storage.py

```diff
@@ -0,0 +1,123 @@
+"""
+Storage consistency checks
+"""
+import json
+import logging
+from uuid import uuid4
+
+from cloud_blobstore import BlobNotFoundError
+
+from dss import Config, Replica
+from dss.storage.hcablobstore import compose_blob_key
+from dss.operations import dispatch
+from dss.operations.util import CommandForwarder, map_bucket
+from dss.events.handlers.sync import dependencies_exist
+
+
+logger = logging.getLogger(__name__)
+
+
+class StorageOperationHandler:
+    def __init__(self, argv, args):
+        self.keys = args.keys.copy() if args.keys else None
+        self.entity_type = args.entity_type
+        self.job_id = args.job_id
+        if args.replica:
+            self.replica = Replica[args.replica]
+            self.handle = Config.get_blobstore_handle(self.replica)
+
+    def forward_command_to_lambda(self, argv, args):
+        cmd_template = f"{argv[0]} {argv[1]}"
+        if "entity_type" in args.__dict__.keys() and "keys" in args.__dict__.keys():
+            del args.entity_type
+        for argname, argval in args.__dict__.items():
+            argname = argname.replace("_", "-")
+            if argname not in ["forward-to-lambda", "keys", "func"]:
+                cmd_template += f" --{argname} {argval}"
+        assert "{}" not in cmd_template
+        cmd_template += " --keys {}"
```

Comment:
There's a user option for keys, but it appears to be hard-coded?

Reply:
Added a docstring to this method, hopefully answering your questions.

Reply:
Meaning, there is a user option for it, but setting this option has different behavior when applied locally, and via lambda that's not mentioned in the option description.

Reply:
I'm not sure I understand?

Check two keys locally:
```
scripts/dss-ops.py storage verify-file-blob-metadata --replica $replica --keys $key1 $key2
```
Check two keys via Lamba:
```
scripts/dss-ops.py storage verify-file-blob-metadata --replica $replica --keys $key1 $key2 --forward-to-lambda
```

Check entire bucket locally:
```
scripts/dss-ops.py storage verify-file-blob-metadata --replica $replica --forward-to-lambda
```

Check entire bucket via Lambda:
```
scripts/dss-ops.py storage verify-file-blob-metadata --replica $replica --forward-to-lambda
```

Reply:
I see now.  Can those examples be put into either a docstring or the argparse menu description?